test/integration/patients/worklist/worklist.js

```diff
@@ -3806,7 +3783,9 @@ context('worklist page', function() {
       .as('patientSidebar')
       .find('.worklist-patient-sidebar__patient-name')
       .should('contain', 'Test Patient')
-      .click();
+      .click()
+      .wait('@routePatientField')
+      .wait('@routePrograms');
```

Comment:
there are a few instances where we need to wait on routes loaded by a `routesFor*` command.  I don't love this, but not doing so makes these tests flaky.